content/doc/developer/publishing/source-code-hosting.adoc

```diff
@@ -0,0 +1,11 @@
+---
+title: Source Code Hosting
+layout: developersection
+---
+
+The Jenkins project only publishes free and open source plugins distributed under an OSI-approved license.
+
+The canonical source code repository is in the +jenkinsci+ GitHub organization to ensure source code access and project continuity in case previous maintainers move on.
+Some plugins we distribute are currently maintained outside the +jenkinsci+ GitHub organization for historical reasons, but this is discouraged.
+
+Plugin maintainers typically have repository admin permissions for plugins they maintain, but some features (like transfer and deletion of repos) are disabled.
```

Comment:
"But we strongly recommend using JIRA for consistency with other plugins and Jenkins components" (GitHub issues makes it harder to link issues between themselves, for instance, in the global Jenkins JIRA instance)

Reply:
Unrelated to source code hosting, will be brought up in the issue tracker section.